app/src/main/res/layout/loading_page.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename this file from loading_page.xml to splash_screen.xml

Reply:
I just did. Thank you!